Framework/CMakeLists.txt

```diff
@@ -100,14 +118,19 @@ add_library(O2QualityControl
   src/WorkflowType.cxx
   src/TimekeeperFactory.cxx
   src/RootFileStorage.cxx
-  src/ReductorHelpers.cxx)
+  src/ReductorHelpers.cxx
+  src/KafkaPoller.cxx)
 
 target_include_directories(
   O2QualityControl
   PUBLIC $<INSTALL_INTERFACE:include>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  ${MODERNCPPKAFKA_ROOT}
+  $<BUILD_INTERFACE:${RDFKAFKA_INCLUDE}>
   PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>)
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
+  ${MODERNCPPKAFKA_ROOT}
```

Comment:
It seems that you have twice `${MODERNCPPKAFKA_ROOT}`

Reply:
you are correct, thanks